pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -459,7 +459,24 @@ func (r *ReconcileInfinispan) secretForUser(user string, realm string, m *infini
 			return nil, err
 		}
 	}
+	return r.createSecretForAuthentication(user, realm, m, reqLogger)
+}
 
+func (r *ReconcileInfinispan) secretForMgmt(user string, realm string, m *infinispanv1.Infinispan, reqLogger logr.Logger) (*corev1.Secret, error) {
```

Comment:
Hmmmm, seems to me that adding another method is a bit too much here. 

Couldn't we just make `secretForUser` take an extra parameter? For management use, the parameter value would be `m.Spec.Management.Authentication.SecretName`. For application user `m.Spec.Connector.Authentication.SecretName`?

Reply:
I choosed that way because I don't like a lot the idea of passing `m` ad also a substruct of the same object `m.Spec...` in the same function call.

Reply:
ok, removed dup code